packages/soxr/test_soxr.py

```diff
@@ -0,0 +1,8 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["soxr-tests", "pytest"])
+def test_soxr(selenium):
+    import pytest
+
+    assert pytest.main(["--pyargs", "soxr", "-k", "not test_multithread"]) == 0
```

Comment:
Let's write a simple smoke test instead of running a full test suite. We prefer not to run a full package test suite as it increases the test running time.